Documentation/Books/Manual/Installation/MacOSX.md

```diff
@@ -1,80 +1,100 @@
 Installing ArangoDB on MacOS
 ============================
 
-The preferred method for installing ArangoDB under MacOS is
-[_homebrew_](#homebrew). However, in case you are not using _homebrew_, we
-provide a [command-line app](#command-line-app) or [graphical
-app](#graphical-app) which contains all the executables.
+ArangoDB under MacOS can be installed via:
 
-{% hint 'info' %} Starting from version 3.4.0 in addition to 
-_homebrew_ and the _dmg_ package a _tar.gz_ archive is available. 
+1. [Homebrew](#homebrew)
+2. [_DMG_ Package](#package-installation)
+3. [_tar.gz_ Archive](#installing-using-the-archive) (starting from v.3.4.0)
+
+{% hint 'info' %}
+When installing ArangoDB via the MacOS package manager Homebrew,
+only the Community Edition is available.
 {% endhint %}
 
 Homebrew
 --------
 
+{% hint 'warning' %}
+The Homebrew installation is updated a few days after the
+official release of a new version.
+{% endhint %}
+
 If you are using [_homebrew_](http://brew.sh/),
 then you can install the latest released stable version of ArangoDB using *brew* as follows:
 
-    brew install arangodb
+```
+brew install arangodb
+```
 
 This will install the current stable version of ArangoDB and all
 dependencies within your Homebrew tree. Note that the server will be
 installed as:
 
-    /usr/local/sbin/arangod
+```
+/usr/local/Cellar/arangodb/3.4.0/sbin/arangod
+```
 
-You can start the server by running the command `/usr/local/sbin/arangod &`.
+You can start the server by running the command `/usr/local/Cellar/arangodb/3.4.0/sbin/arangod &`.
 
 Configuration file is located at
 
     /usr/local/etc/arangodb3/arangod.conf
 
 The ArangoDB shell will be installed as:
 
-    /usr/local/bin/arangosh
+```
+/usr/local/Cellar/arangodb/3.4.0/bin/arangosh
+```
 
 You can uninstall ArangoDB using:
 
-    brew uninstall arangodb
+```
+brew uninstall arangodb
+```
 
 However, in case you started ArangoDB using the _launchctl_, you
 need to unload it before uninstalling the server:
 
-    launchctl unload ~/Library/LaunchAgents/homebrew.mxcl.arangodb.plist
+```
+launchctl unload ~/Library/LaunchAgents/homebrew.mxcl.arangodb.plist
+```
 
 Then remove the LaunchAgent:
 
-    rm ~/Library/LaunchAgents/homebrew.mxcl.arangodb.plist
+```
+rm ~/Library/LaunchAgents/homebrew.mxcl.arangodb.plist
+```
 
-**Note**: If the latest ArangoDB Version is not shown in homebrew, you
-also need to update homebrew:
-
-    brew update
+{% hint 'tip' %}
+If the latest ArangoDB version is not shown in Homebrew, you
+also need to update Homebrew executing the command `brew update`.
```

Comment:
please add something along the line: 
We push new ArangoDB Versions directly to Homebrew; However it may take a bit for a PR to be accepted.